README.mediawiki

```diff
@@ -623,13 +623,6 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Christian Decker, Anthony Towns
 | Standard
 | Draft
-|-
-| [[bip-0119.mediawiki|119]]
-| Consensus (soft fork)
-| CHECKTEMPLATEVERIFY
-| Jeremy Rubin, James O'Beirne
-| Standard
-| Draft
```

Comment:
Don't remove. Rather use `Withdrawn`.

Reply:
What is your purpose then? If a BIP is in `Draft` state and it will be cancelled, it will be `Withdrawn`.